data/categories.txt

```diff
@@ -1807,6 +1807,21 @@ el:^Υπόγειος|σωλήνας|μετρό|υπόγεια|μεταφορές
 sk:3^Metro|podzemná doprava
 sw:^Reli ya chini
 
+railway-level_crossing
+en:^Level Crossing
+ru:^Железнодорожный переезд
+fr:^Passage à niveau
+de:^Bahnübergang
+it:^Passaggio a livello
+es:^Paso a nivel
+pt:^Passagem de nível
+pl:^Przejazd kolejowy
+cs:^Železniční přejezd
+nl:^Overweg
+da:^Jernbaneoverskæring
+fi:^Tasoristeys
+hu:^Útátjáró
```

Comment:
What about other languages?

Reply:
What are your contribution guidelines regarding translations? What languages are at least necessary?

Reply:
The only necessary language is English, but we try to add as many as possible at least in the simple situations such as categories.txt where Grammar is not needed.
Here is the list: https://github.com/mapsme/omim/blob/master/indexer/categories_holder.cpp#L152